src/app/app.routes.ts

```diff
@@ -1,3 +1,9 @@
 import { Routes } from '@angular/router';
+import { CharactersComponent } from '../app/features/characters/characters.component';
```

Comment:
the same here - _path_ for _features_. This will help you to avoid using ../app/features and use just _@features_ anywhere

Reply:
I will fix it in the next PR